config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -25,8 +25,7 @@ spec:
     The Windows Machine Config Operator configures Windows Machines into nodes, enabling Windows container workloads to
     be run on OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
     or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/latest/windows_containers/configuring-byoh-windows-instance.html)
-    Through either method, the Windows instance to be configured must have the Docker container runtime installed. The operator
-    completes all the necessary steps to configure the underlying VM so that it can join the cluster as a worker node.
+    The operator completes all the necessary steps to configure the underlying VM so that it can join the cluster as a worker node.
```

Comment:
```suggestion
    The operator completes all the necessary steps to configure the Windows instance so that it can join the cluster as a worker node.
```